Gems/MiniAudio/Code/Include/MiniAudio/MiniAudioListenerBus.h

```diff
@@ -18,8 +18,21 @@ namespace MiniAudio
     public:
         ~MiniAudioListenerRequests() override = default;
 
-        virtual void SetFollowEntity(AZ::EntityId followEntity) = 0;
+        virtual void SetFollowEntity(const AZ::EntityId followEntity) = 0;
         virtual void SetPosition(const AZ::Vector3& position) = 0;
+        virtual AZ::u32 GetChannelCount() const = 0;
+
+        //! Cone controls for directional attenuation
+        virtual float GetInnerAngleInRadians() const = 0;
+        virtual void SetInnerAngleInRadians(const float innerAngleInRadians) = 0;
+        virtual float GetInnerAngleInDegrees() const = 0;
+        virtual void SetInnerAngleInDegrees(const float innerAngleInDegrees) = 0;
+        virtual float GetOuterAngleInRadians() const = 0;
+        virtual void SetOuterAngleInRadians(const float outerAngleInRadians) = 0;
+        virtual float GetOuterAngleInDegrees() const = 0;
+        virtual void SetOuterAngleInDegrees(const float outerAngleInDegrees) = 0;
+        virtual float GetOuterVolume() const = 0;
+        virtual void SetOuterVolume(const float outerVolume) = 0;
```

Comment:
We don't usually enforce const usage of types that are passed by value for listeners (unless they're references) as it needlessly constrains whoever implements the EBus handler. e.g. if the implementer chooses to modify `followEntity` or `innerAngleInRadians` inside their function implementation they can do so without affecting any code outside their implementation.

```suggestion
        virtual void SetFollowEntity(AZ::EntityId followEntity) = 0;
        virtual void SetPosition(const AZ::Vector3& position) = 0;
        virtual AZ::u32 GetChannelCount() const = 0;

        //! Cone controls for directional attenuation
        virtual float GetInnerAngleInRadians() const = 0;
        virtual void SetInnerAngleInRadians(float innerAngleInRadians) = 0;
        virtual float GetInnerAngleInDegrees() const = 0;
        virtual void SetInnerAngleInDegrees(float innerAngleInDegrees) = 0;
        virtual float GetOuterAngleInRadians() const = 0;
        virtual void SetOuterAngleInRadians(float outerAngleInRadians) = 0;
        virtual float GetOuterAngleInDegrees() const = 0;
        virtual void SetOuterAngleInDegrees(float outerAngleInDegrees) = 0;
        virtual float GetOuterVolume() const = 0;
        virtual void SetOuterVolume(float outerVolume) = 0;
```

Reply:
This is a good point, I actually intended for these to be passed by reference.